### PR TITLE
Update GovCloud migration process info for outside-TTS use

### PR DESCRIPTION
**Pull request by brittag**
**Don't merge yet.** This needs to be updated with the upcoming link to self-invitation for GovCloud access.

These changes are meant to help non-18F people migrate their orgs to GovCloud, to support [this message](https://docs.google.com/document/d/1l-akEeE2cRxML_svuQTmo__naN80sN-Jj_n6Jqt9DoE/edit#heading=h.jgcxtjglazl) and related messaging. Feel free to add more improvements to this WIP branch - this is just a start to help.

Changes:

* Start with getting access to GovCloud.
* Provide alternative to org request form for people outside TTS.
* Remove links to incomplete info about service migration.
* Remove "unresolved details" chunk.
* Include the updated text from https://github.com/18F/cg-site/pull/763 to avoid merge conflicts.

### CODE REVIEW

**Comment by brittag** [comment]
(To be clear about the WIP-ness of this - I don't currently have plans to work on this more - would like to hand it to migrators for finishing it up. :) )

**Comment by brittag** [comment]
This depends on https://github.com/18F/cg-uaa-extras/pull/81 for the link to self-invitation.

File: content/docs/apps/govcloud.md
Comment (fureigh) [inline content/docs/apps/govcloud.md]: Should they also specify the quota they'd like, or does the self-invitation process include that?

Comment (fureigh) [reply content/docs/apps/govcloud.md]: @jcscottiii @rememberlenny [Self-invitation](https://account.fr.cloud.gov/signup) is now a thing (:tada:)... is requesting a creation of an org still a manual process that requires them to email us the org name and desired quota, though? Asking so I can update this step.

File: content/docs/apps/govcloud.md
Comment (fureigh) [inline content/docs/apps/govcloud.md:86]: @jcscottiii @rememberlenny Are these non-18F/TTS instructions accurate?

**Review by fureigh — CHANGES_REQUESTED** [review]
I think this is good to go, pending confirmation that the flagged non-18F/TTS instructions are accurate.

File: content/docs/apps/govcloud.md
Comment (brittag) [reply content/docs/apps/govcloud.md]: Yes, creating an org is still a manual process they should email us about. Self-invitation is unrelated - it's just about access to the GovCloud environment.

File: content/docs/apps/govcloud.md
Comment (jcscottiii) [reply content/docs/apps/govcloud.md:86]: Would like to make sure they tag their subject line with something like `[GovCloud Org Creation]` to ensure we don't lose track. Besides that, seems good

Comment (fureigh) [reply content/docs/apps/govcloud.md:86]: @jcscottiii Thanks! Do we need them to specify a quota too or nah?

Comment (jcscottiii) [inline content/docs/apps/govcloud.md]: nahh

Comment (jcscottiii) [reply content/docs/apps/govcloud.md:86]: nahh

**Review by jcscottiii — APPROVED** [review]

File: content/docs/apps/govcloud.md
Comment (mogul) [inline content/docs/apps/govcloud.md]: You can provide a default body for the `mailto:` URL that prompts them for any necessary information as they fill out the mail.

File: content/docs/apps/govcloud.md
Comment (fureigh) [reply content/docs/apps/govcloud.md:86]: I'll move forward on merging this, but: how/when will we get this info from them? Or do they just get set up with a default?

File: content/docs/apps/govcloud.md
Comment (fureigh) [reply content/docs/apps/govcloud.md]: Good call. Seems like there may be further changes to what constitutes the necessary information, so I'll merge this as-is and can start a PR with a minimal version.

Comment (fureigh) [reply content/docs/apps/govcloud.md]: ...ah, this request is blocking the merge. Commit TK, then.

**Review by fureigh — APPROVED** [review]

**Pull request merged**
